core/server/xendorser/plugin-default/xendorser_impl.go

```diff
@@ -5,21 +5,34 @@ import (
 	"errors"
 	"io/ioutil"
 
-	context "golang.org/x/net/context"
+	"golang.org/x/net/context"
```

Comment:
Switch to the `context` package of the standard library

Reply:
No change now